src/Microsoft.Diagnostics.Runtime/src/Utilities/Platform/LinuxFunctions.cs

```diff
@@ -96,6 +95,7 @@ public LinuxFunctions()
 
         internal static void GetVersionInfo(IDataReader dataReader, ulong baseAddress, ElfFile loadedFile, out VersionInfo version)
         {
+            Console.WriteLine($"GetVersionInfo2");
```

Comment:
Is this intentional?

Reply:
Console.Write anywhere in the main library is never intentional.  I probably shouldn't be writing code with 3 hours sleep...

Fixed in https://github.com/microsoft/clrmd/pull/458.